src/file.rs

```diff
@@ -33,68 +33,75 @@ pub struct FileLoggerBuilder {
 }
 
 impl FileLoggerBuilder {
-    /// Makes a new `FileLoggerBuilder` instance.
+	/// Makes a new `FileLoggerBuilder` instance.
```

Comment:
It seems that the indentations of this file are broken. Could you re-format this file by using `rustfmt` (i.e., `$ cargo fmt`)?

Reply:
done